test/.kontena_client.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I don't think this belongs here

Reply:
@kke, yup - I will remove that one, thanks for the catch! 😄